src/Pelagos/Bundle/AppBundle/Command/SetPrimaryPointOfContactCommand.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace Pelagos\Bundle\AppBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+use Pelagos\Entity\Dataset;
+use Pelagos\Entity\DatasetSubmission;
+use Pelagos\Entity\PersonDatasetSubmission;
+
+/**
+ * Set Primary Point of Contact for Accepted Datasets.
+ *
+ * @see ContainerAwareCommand
+ */
+class SetPrimaryPointOfContactCommand extends ContainerAwareCommand
+{
+    /**
+     * Configures the current command.
+     *
+     * @return void
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('dataset:set-primary-point-of-contact')
+            ->setDescription('Set primary point of contact for datasets which do not have a primary');
+    }
+
+    /**
+     * Executes the current command.
+     *
+     * @param InputInterface  $input  An InputInterface instance.
+     * @param OutputInterface $output An OutputInterface instance.
+     *
+     * @return void
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $entityManager = $this->getContainer()->get('doctrine.orm.entity_manager');
+
+        $datasets = $entityManager->getRepository(Dataset::class)
+            ->findBy(array('datasetStatus' => Dataset::DATASET_STATUS_ACCEPTED));
```

Comment:
only the Accepted??? 
What about InReview?

Reply:
This data discrepancy happened during the release of 5.0.0, so all the datasets which were in accepted status got migrated without a primary contact. And checking for InReview status and setting an attribute without creating a new dataset submission object might bring new complications, and there should be only 2-3 datasets in that InReview status which don't have a primary contact.